packages/liveblocks-client/src/types.ts

```diff
@@ -123,27 +123,24 @@ export type AuthenticationToken = {
   info?: any;
 };
 
+//
+// NOTE: The `count` and `toArray()` methods are here for backward-compatible
+// reasons only. We want to eventually deprecate these APIs in favor of
+// `length` and just accessing the array directly here.
+//
+type ReadonlyArrayish<T> = readonly T[] & {
```

Comment:
What do you think of the name: `ReadonlyArrayWithLegacyMethods` ?

Reply:
Good suggestion, adopted in b6d6990.